sbi/inference/snpe/snpe_base.py

```diff
@@ -435,6 +423,25 @@ def _train(
         self._summary["epochs"].append(epoch)
         self._summary["best_validation_log_probs"].append(self._best_val_log_prob)
 
+    def _loss(self, round_idx: int, theta: Tensor, x: Tensor, masks: Tensor) -> Tensor:
+        """Return the right calibrated loss, depending on the round.
+
+        The loss is the negative log prob. Irrespective of the round or SNPE method
+        (A, B, or C) it weighted with the calibration kernel.
+
+        Returns:
+            calibration kernel weighted negative log prob.
+        """
+
+        if round_idx == 0:
+            # Use plain posterior log prob for first round.
```

Comment:
There must be something more descriptive to say than plain... (sorry no suggestion).